spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/DefaultRestTemplateConfig.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2013-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.netflix.eureka;
+
+import java.util.Objects;
+
+public class DefaultRestTemplateConfig implements RestTemplateConfig {
```

Comment:
Please add javadocs for any new classes. Please add `@since 4.0.0` to the javadocs of new classes. Please add your full name and surname with `@author` tag to the javdocs of any added/ edited classes.